crates/openapi/src/routes/merchant_connector_account.rs

```diff
@@ -60,9 +61,70 @@
 )]
 pub async fn connector_create() {}
 
+#[cfg(feature = "v2")]
+#[utoipa::path(
+    post,
+    path = "/connector_accounts",
+    request_body(
+        content = MerchantConnectorCreate,
+        examples(
+            (
+                "Create a merchant connector account with minimal fields" = (
+                    value = json!({
+                        "connector_type": "fiz_operations",
```

Comment:
Can we use `payment_processor` instead in all the examples (even the existing ones)?